src/test/scala/it/pagopa/interop/purposeprocess/SpecData.scala

```diff
@@ -43,28 +43,16 @@ object SpecData {
     agreementApprovalPolicy = CatalogManagement.AgreementApprovalPolicy.AUTOMATIC
   )
 
-  val institution: PartyManagement.Institution = PartyManagement.Institution(
-    id = UUID.randomUUID(),
-    origin = "yadayada",
-    originId = "yadayada",
-    externalId = "yadayada",
-    description = "Organization description",
-    digitalAddress = "address",
-    address = "address",
-    taxCode = "taxCode",
-    attributes = Seq.empty,
-    zipCode = "00000",
-    institutionType = "PUBLIC"
-  )
-
   val tenant: Tenant = Tenant(
     UUID.randomUUID(),
-    selfcareId = institution.id.toString.some,
+    selfcareId = UUID.randomUUID.toString().some,
```

Comment:
```suggestion
    selfcareId = UUID.randomUUID.toString.some,
```

Reply:
This was the exact problem with my tests. UUID was invalid and in my mind that was related to the way I made changes inside the codebase (when I was passing tenant.name inside Organization). Thanks again!